src/intrinsics/ecma262/Date.js

```diff
@@ -189,7 +189,13 @@ export default function (realm: Realm): NativeFunctionValue {
 
   // ECMA262 20.3.3.2
   func.defineNativeMethod("parse", 1, (context, [string]) => {
-    throw new Error("TODO: Implement Date.parse");
+    let concreteString = string.throwIfNotConcrete();
+    let s = ToString(realm, concreteString);
+    if ((s.length > 0 && s.charAt(s.length - 1) !== 'Z') ||
```

Comment:
Do you mean the check for whether the string is concrete? Not quite sure what you mean :(. Also, for the spelling check, you mean to add a test that would fail if strictlyMonotonicDateNow is now true (i.e.: it was misspelled)?